ios/TwilioVoiceReactNative+CallKit.m

```diff
@@ -36,7 +36,7 @@ - (void)initializeCallKit {
 - (void)reportNewIncomingCall:(TVOCallInvite *)callInvite {
     self.callInviteMap[callInvite.uuid.UUIDString] = callInvite;
     
-    // Frontline specific logic
+    // Twilio Frontline specific logic
```

Comment:
@kbagchiGWC @mhuynh5757 @charliesantos 
While working on the iOS walkthrough, I found some logic that was added very specifically for the Twilio Frontline app. I am thinking that we keep a branch for Frontline to reference in their pipeline (IIRC they are referencing us by source, not by NPM). We can update this branch if needed. We can then remove these internal logics before the release.